packages/editor/src/components/rich-text/index.js

```diff
@@ -354,6 +347,12 @@ export class RichText extends Component {
 		if ( unstableOnFocus ) {
 			unstableOnFocus();
 		}
+
+		document.addEventListener( 'selectionchange', this.onSelectionChange );
+	}
+
+	onBlur() {
+		document.removeEventListener( 'selectionchange', this.onSelectionChange );
```

Comment:
I guess `blur`, despite bubbling, doesn't fire when moving between elements in the same `contenteditable` ?

https://codepen.io/aduth/pen/VVqOmq?editors=1111

(In other words, I suspected there could be an issue, but it doesn't appear so)

Reply:
No, it shouldn't, it's starts at the `contenteditable` element.